### DISCUSSION TITLE
Invalid Discord Invite - figma-to-react-native

### DISCUSSION THREAD

**Discussion by witcher-shailesh** [kat-tax/ult#46] [General]
Hi @TheUltDev,

Firstly, I want to express my appreciation for the effort you've put into building figma-to-react-native. It's an excellent tool for anyone starting their journey with React Native.

I wanted to join the Discord community, but it seems the invite URL is invalid. Is there a specific reason for this?

**Comment by TheUltDev** [comment]
@witcher-shailesh thank you for the kind words

Also thank you for pointing that out, that's... very unfortunate to say the least.
I've updated the link: https://discord.kat.tax

**Reply by witcher-shailesh** [reply]
Somehow it's still not working for me 😞 
<img width="1162" alt="Screenshot 2024-09-13 at 12 36 43 PM" src="https://github.com/user-attachments/assets/87221556-0d38-43eb-888f-018bd3a6e913">

**Reply by TheUltDev** [reply]
@witcher-shailesh here's a custom one: https://discord.gg/cKuFhe97mW
